input/docs/writing-builds/preprocessor-directives/tool/nuget-provider.md

```diff
@@ -0,0 +1,101 @@
+Order: 20
+Title: nuget Provider
+Description: Package installer for installing tools packaged as NuGet packages
+---
+
+Package installer for installing tools packaged as NuGet packages.
+
+The following URI parameters are supported by the `nuget` package installer.
+
+# Source
+
+This is not a named parameter, but it is permitted as per the URI definition.
+By default, the provider will attempt to install tools from nuget.org.
+If the package is hosted on another feed the installation source can be overridden.
+
+## Example
+
+Download from myget.org:
+
+```csharp
+#tool nuget:https://myget.org/f/Cake/?package=Cake.Foo
+```
+
+Install tool from a local directory:
+
+```csharp
+#tool nuget:file://localhost/packages/?package=Cake.Foo
+```
+
+# Package
+
+The name of the NuGet package that should be installed.
+
+## Example
+
+```csharp
+#tool nuget:?package=Cake.Foo
+```
+
+# Version
+
+The specific version of the .NuGet package that should be installed.
+If not provided, the latest version package that is available will be installed.
+
+## Example
+
+```csharp
+#tool nuget:?package=Cake.Foo&version=1.2.3
+```
+
+# Prerelease
+
+To install prerelease tools the `prerelease` parameter needs to be passed.
```

Comment:
```suggestion
To install prerelease tools without defining a specific `version` (i.e. latest prerelease version available), the `prerelease` parameter needs to be passed.
```

I believe the `prerelease` is optional since Cake v1.0.0, when the user specifies a version that is pre-release e.g. `#tool nuget:?package=Cake.Foo&version=1.2.3-beta`.